src/main/323C.c

```diff
@@ -326,7 +326,71 @@ INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80014A90);
 
 INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80014C70);
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80014DC4);
+void func_80014DC4(void)
+{
+    u8 var_v1;
+
+    D_80137DFC = 0;
+
+    for (var_v1 = 0; var_v1 < 6; var_v1++) {
+        D_8013E198[var_v1] = -1;
+    }
+    for (var_v1 = 0; var_v1 < 4; var_v1++) {
+        D_8013E1C8[var_v1] = -1;
+    }
+    for (var_v1 = 0; var_v1 < 4; var_v1++) {
+        D_8013924C[var_v1] = -1;
+    }
+    for (var_v1 = 0; var_v1 < 24; var_v1++) {
+        D_80139234[var_v1] = -1;
+    }
+
+    D_8013DC10 = &_SsNoteOn;
```

Comment:
Thinking this is `_SsFCALL` https://github.com/sozud/mmx4/blob/b1dbad89d772b280a6fac8dbb91e854bc258c099/include/psy-q-4.0/LIBSND.H#L321

See this excerpt https://github.com/Zackmon/lunar2-psx-decomp/blob/3798b47d77a4e4e8008102345ba1cd6aa0f661bb/include/psyq4.6/libsnd.h#L464